pkg/restapi/restapi_test.go

```diff
@@ -45,3 +45,25 @@ func TestNew_Success(t *testing.T) {
 	require.NotEmpty(t, controller.GetOperations())
 
 }
+
+//func TestPOST(t *testing.T) {
```

Comment:
can we remove this instead of commenting, if not required ? else can we add a TODO ?